### PR TITLE
Add new whitespace lints to censys-definitions

### PR DESCRIPTION
**Pull request by kumarde**

### CODE REVIEW

File: proto/zlint.proto [hunk 1]
```diff
@@ -236,5 +236,9 @@ message Lints {
     LintResult e_subject_state_name_max_length = 191;
     LintResult w_multiple_subject_RDN = 192;
     LintResult w_multiple_issuer_RDN = 193;
+    LintResult w_issuer_dn_trailing_whitespace = 194;
+    LintResult w_issuer_dn_leading_whitespace = 195;
+    LintResult w_subject_dn_trailing_whitespace = 196;
+    LintResult w_subject_dn_leading_whitespace = 197;
     // highest ID in use is currently 193
```
Comment (dadrian) [inline proto/zlint.proto]: Comment should be bumped to 197

Comment (zakird) [inline proto/zlint.proto]: @kumarde yo.

**Comment by zakird** [comment]
@kumarde please take care of this.

File: proto/zlint.proto (hunk 1 above)
Comment (kumarde) [inline proto/zlint.proto]: done.

**Comment by zakird** [comment]
This looks fine. Please make sure these are in schema.

**Pull request merged**